profiles/car.lua

```diff
@@ -343,15 +344,19 @@ function way_function (way, result)
   -- local barrier = way:get_value_by_key("barrier", "")
   -- local cycleway = way:get_value_by_key("cycleway", "")
   local service = way:get_value_by_key("service")
+  local destination = get_destination(way)
 
   -- Set the name that will be used for instructions
   local has_ref = ref and "" ~= ref
   local has_name = name and "" ~= name
+  local has_destination = destination and "" ~= destination
 
   if has_name and has_ref then
     result.name = name .. " (" .. ref .. ")"
   elseif has_ref then
     result.name = ref
+  elseif has_name and has_destination then
+    result.name = name .. " (" .. destination .. ")"
```

Comment:
- [ ] Do the same for foot / walk profile

I only now realized I hooked the lib function in only for car.